aws/resource_aws_db_parameter_group.go

```diff
@@ -295,6 +295,26 @@ func resourceAwsDbParameterGroupUpdate(d *schema.ResourceData, meta interface{})
 			}
 			d.SetPartial("parameter")
 		}
+
+		// Reset parameters that have been removed
+		parameters, err = expandParameters(os.Difference(ns).List())
```

Comment:
I'm seeing in the `ResetDBParameterGroupInput` SDK documentation [here](https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#ResetDBParameterGroupInput) that there is a documented 20 item max, so it would probably be best to add that logic in here. There's an example of doing this with a for-loop [here](https://github.com/terraform-providers/terraform-provider-aws/pull/10161/files#diff-497d6ad0ec0d61809a57fc6ff6ffaadbR280-R281). 🙂